GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -2662,26 +2663,8 @@ bool FramebufferManagerCommon::GetDepthbuffer(u32 fb_address, int fb_stride, u32
 
 	bool flipY = (GetGPUBackend() == GPUBackend::OPENGL && !useBufferedRendering_) ? true : false;
 
-	bool retval;
-	if (true) {
-		// Always use ReadbackDepthbufferSync (while we debug it)
-		buffer.Allocate(w, h, GPU_DBG_FORMAT_16BIT, flipY);
-		retval = ReadbackDepthbufferSync(vfb->fbo, 0, 0, w, h, (uint16_t *)buffer.GetData(), w, w, h);
-	} else {
-		// Old code
-		if (gstate_c.Use(GPU_SCALE_DEPTH_FROM_24BIT_TO_16BIT)) {
-			buffer.Allocate(w, h, GPU_DBG_FORMAT_FLOAT_DIV_256, flipY);
-		} else {
-			buffer.Allocate(w, h, GPU_DBG_FORMAT_FLOAT, flipY);
-		}
-		// No need to free on failure, that's the caller's job (it likely will reuse a buffer.)
-		retval = draw_->CopyFramebufferToMemory(vfb->fbo, Draw::FB_DEPTH_BIT, 0, 0, w, h, Draw::DataFormat::D32F, buffer.GetData(), w, Draw::ReadbackMode::BLOCK, "GetDepthBuffer");
-		if (!retval) {
-			// Try ReadbackDepthbufferSync, in case GLES.
-			buffer.Allocate(w, h, GPU_DBG_FORMAT_16BIT, flipY);
-			retval = ReadbackDepthbufferSync(vfb->fbo, 0, 0, w, h, (uint16_t *)buffer.GetData(), w, w, h);
-		}
-	}
+	buffer.Allocate(w, h, GPU_DBG_FORMAT_16BIT, flipY);
+	bool retval = ReadbackDepthbuffer(vfb->fbo, 0, 0, w, h, (uint16_t *)buffer.GetData(), w, w, h, Draw::ReadbackMode::BLOCK);
```

Comment:
For debugging (which is what this function is for), it's sometimes nice to see the actual values in the buffer.  It's useful to see what it's rounding from.  Otherwise when a `>=` test fails or there's z-fighting, it's hard to tell why - it looks like it shouldn't have happened, sometimes.

-[Unknown]

Reply:
Sure, but how does this change make it worse? Can just jump into ReadbackDepthBuffer and look at the values there?

Reply:
I'm talking about in the GE debugger when examining frames, not using a VS debugger and looking at the bytes or something.

-[Unknown]

Reply:
But I don't get it - what functionality are we losing here?

Reply:
For example, someone reports a bug that some scene is rendering wrong.  There can be hundreds of causes.  I take a look at the frame dump and start stepping through.

Before: as I'm puzzling over why something should or shouldn't draw, I can hover over pixels in the depth buffer and see that depth was 12345.67 and the test is <= 12345.  Oh, that's interesting.

Now: why should anyone care?  The GE debugger says 12345, which is perfect.  Mystery deepens.  Maybe if I don't find anything I'll look at renderdoc and it can help me.  Things just get slower.

-[Unknown]

Reply:
Oh! Your point is that we lost precision. Doh. Yeah.. You're right. I'll revert this part.